.rubocop.yml

```diff
@@ -13,4 +10,10 @@ Rails/ActionFilter:
   EnforcedStyle: filter
 
 Metrics/MethodLength:
-  Max: 20
\ No newline at end of file
+  Max: 20
+
+Metrics/LineLength:
+  Max: 100
```

Comment:
Reducing this increases the number of offences due to some lines in engine.rb, can you fix them?